packages/nc-gui/components/cell/SingleSelect/utils.ts

```
@@ -0,0 +1,53 @@
+import type { ColumnType, SelectOptionType } from 'nocodb-sdk'
+
+export const getOptions = (
+  column: ColumnType,
+  isEditColumn: boolean,
+  isForm: boolean,
+): (SelectOptionType & { value?: string })[] => {
+  if (column && column?.colOptions) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)